lib/processors/versionInfoGenerator.js

```diff
@@ -14,6 +16,423 @@ function getTimestamp() {
 	return year + month + day + hours + minutes;
 }
 
+/**
+ * Manifest libraries as defined in the manifest.json file
+ *
+ * @typedef {object<string, object>} ManifestLibraries
+ *
+ *  * @example
+ * {
+ *   sap.chart: {
+ *       lazy: true
+ *   },
+ *   sap.f: { }
+ * }
+ */
+
+/**
+ * Extracted information from a manifest's <code>sap.app</code> and <code>sap.ui5</code> sections.
+ */
+class ManifestInfo {
```

Comment:
IMO, the ManifestInfo is not of much use in its current implementation. it just requires copying back and forth data and brings no added value.

Reply:
used typedef instead with 1fe9c25